### PR TITLE
fix

### LINKED ISSUE #9: fix: Buttons and hyperlink fixes
- Hyperlink of Login Ecom.
- Size of Landing page CTA button.
- link logout to home(redirect)
- card btn alignment in wishlist page.
- Filter selection issue.
- Change in Cart button text.

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
This PR fixes #9
- fixed navigation hyperlinks.
- fixed the alignments of buttons of card in product listing page.
- fixed button text in Cart

### CODE REVIEW

File: Cart/cart.css [hunk 1]
```diff
@@ -70,14 +70,14 @@
 }
 
 .checkout-container {
-    border: 1px solid black;
+    border: 1px solid var(--prime-blue);
     padding: 2rem;
     width: 400px;
```
Comment (RakshithBhat03) [inline Cart/cart.css]: You can use `rem` here.

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested a small change, the rest looks good. Good use of variables. Good to merge.

File: Cart/cart.css (hunk 1 above)
Comment (shashidwivedi) [reply Cart/cart.css:75]: For smaller units like 1px, pixel unit is fine

Comment (srejitk) [inline Cart/cart.css]: Would suggest using relative units here.

File: Products/product.css [hunk 2]
```diff
@@ -28,11 +28,24 @@
...
+
+.card-btn {
+    width: 90%;
+}
+
+.action-btns {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (srejitk) [inline Products/product.css]: This can be extracted into a utils class

**Review by shuklahimanshu81 — APPROVED** [review]
As suggested by Rakshith you  could have used rem in checkout-container width. As always your code structure is very clean. Learnt few new things after reading your code. Green light for merge from my side Maruthi.

File: Cart/cart.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply Cart/cart.css]: Thanks for pointing it out, will make the change

File: Products/product.css [hunk 3]
```diff
@@ -28,11 +28,24 @@
 
 .ecom-card {
     width: 25rem;
-    height: 45rem;
+    height: 47rem;
+}
+
+.card-btn {
+    width: 90%;
+}
```
Comment (rkapoor10) [inline Products/product.css:36]: card-btn class doesn't tell you if it's changing the width. you may add some utility classes for widths and use them here.

File: style.css [hunk 4]
```diff
@@ -118,6 +130,9 @@ body {
     position: absolute;
     height: 100%;
     background: transparent;
+    color: var(--secondary-white);
```
Comment (shashidwivedi) [inline style.css:133]: Will suggest to rethink color variables like these. Are you using some naming convention?

**Review by shashidwivedi — APPROVED** [review]
Rest looks good, can be merged

**Review by srejitk — APPROVED** [review]
Looks good. Have given a few suggestion. Would also recommend adding inline comments for better readability.

File: Products/products.html [hunk 5]
```diff
@@ -124,7 +124,7 @@ <h1 class="prod-title padding-m">Katana</h1>
                         <span class="ecom-disc">Save 20%</span>
                     </div>
 
-                    <div class="action-btns flex-center">
+                    <div class="action-btns">
```
Comment (rkapoor10) [inline Products/products.html]: why did you remove flex-center class and gave its properites in action-btns?

File: Products/products.html [hunk 6]
```diff
@@ -142,7 +142,7 @@ <h1 class="prod-title padding-m">Katana</h1>
...
```
Comment (rkapoor10) [inline Products/products.html]: same as above

File: Products/products.html [hunk 7]
```diff
@@ -160,7 +160,7 @@ <h1 class="prod-title padding-m">Katana</h1>
...
```
Comment (rkapoor10) [inline Products/products.html]: same as above

File: style.css (hunk 4 above)
Comment (MaruthiPothuganti) [reply style.css:133]: Not any as of now. WIll refactor down the line

File: Cart/cart.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply Cart/cart.css]: Thanks for the suggestion. Will update with Rem

File: Products/product.css (hunk 2 above)
Comment (MaruthiPothuganti) [reply Products/product.css]: Already have an util class but isn't working. Let me Recheck.

**Review by rkapoor10 — APPROVED** [review]
Looks good to merge

**Comment by MaruthiPothuganti** [comment]
This PR fixes #9

File: Products/product.css (hunk 3 above)
Comment (MaruthiPothuganti) [reply Products/product.css:36]: The parent here is fixed. So the button here just occupies 90% of that. It doesnt change the width.

File: Products/products.html (hunk 5 above)
Comment (MaruthiPothuganti) [reply Products/products.html]: Thanks. Changed it
